guides/back-end-projects.md

```diff
@@ -0,0 +1,104 @@
+---
+explains: The style guide to the way we organize our back-end projects
+---
+
+# How we use docker and docker-compose
+
+A lot of the software we make at City of Amsterdam uses docker containers.
+
+In general we follow the common community guidelines:
+- [Dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/)
+, including multi-stage builds
+
+Preferably use [docker-ce](https://www.docker.com/community-edition)
+and [docker-compose](https://docs.docker.com/compose/), most recent version
+
+.dockerignore files are used to exclude files that are not relevant to the build.
+Ideally this file starts by ignoring everything (`*`) and then explicitly add the files that belong to the build
+by means of exceptions (e.g. `!**\*.py` to include all python source files).
+Most often you will however find .dockerignore files that only specify files that are not part of the build 
+
+# How we use Python
```

Comment:
The HTML spec, and therefor MarkDown only allows one H1 (#) tag on a page, to make sure a page has 1 headline. This H1 is also used by Jekyll to generate the index etc.

Consider breaking these headings out into separate pages if they truly are different sections. Or make them in to lower hierarchy headings.